backend/core/src/applications/services/applications.service.ts

```diff
@@ -103,6 +104,19 @@ export class ApplicationsService {
 
     const result = await qb.getMany()
 
+    const promiseArray = result.map((application) =>
+      this.getDuplicateFlagsForApplication(application.id)
+    )
+    const flags = await Promise.all(promiseArray)
+    result.forEach((application, index) => {
+      if (flags[index].status) {
+        application.markedAsDuplicate = flags[index].status === ApplicationReviewStatus.duplicate
+      } else {
+        application.markedAsDuplicate = false
+      }
+      application.flagged = !!flags[index].id
+    })
```

Comment:
I wonder if this should be done after the check for user authorization so we avoid doing the duplicate db call if the user isn't even authorized. @YazeedLoonat thoughts?

Reply:
just to add some notes here from yesterday's discussion:
I think we do want to do that but for right now its probably okay to leave this as is. 

We more or less need to undertake an auth investigation and this can fall into that investigation bucket